content/influxdb/cloud-dedicated/admin/query-system-data.md

```diff
@@ -359,9 +502,11 @@ SELECT
   SUM(total_l0_bytes) + SUM(total_l1_bytes) + SUM(total_l2_bytes) AS total_bytes
 FROM
   system.compactor
+WHERE
+  table_name = 'TABLE_NAME'
```

Comment:
Maybe also add something like `for a specific table` at the end of the example title -- i.e. `View the total size of all compacted partitions in bytes for a specific table`